ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala

```diff
@@ -107,7 +107,7 @@ final class OngoingStreamAuthIT
       // Making a change to the user Alice
       _ <- grantUserRightsByAdmin(
         userId = userIdAlice,
-        Right(Right.Kind.CanActAs(Right.CanActAs(UUID.randomUUID().toString))),
+        Right(Right.Kind.ParticipantAdmin(Right.ParticipantAdmin())),
```

Comment:
I'd rather we added access to bob's party

Reply:
I dont see the big difference here, as by the spec we seem to break the access on any chance